src/components/Hero/HeroStyles.jsx

```diff
@@ -100,6 +99,10 @@ export const HeroStyles = styled.section`
       font-size: 48px;
     }
 
+    .hero__heading-image {
+      max-width: 100%;
```

Comment:
Fixed in #6 (see comment below)

Reply:
This has not been fixed, it does not resize the image as the page scales down, thus the image goes out of the screen.

Reply:
Can you post screenshot?

Reply:
> 
Current Screenshot of Deploy on Dev:
![image](https://user-images.githubusercontent.com/83686967/143630224-95d5e367-de2b-457d-8ca2-54dc76ef3d84.png)